src/Sensors.hpp

```diff
@@ -52,7 +52,7 @@ class Sensors {
    public:
 
     /// Supported devices. Auto is for Honeywell and Plantower sensors and similars
-    enum SENSOR_TYPE { Auto, Panasonic, Sensirion, Mhz19, CM1106, SCD30co2 };
+    enum SENSOR_TYPE { Auto, Panasonic, Sensirion, SDS011, Mhz19, CM1106, SCD30co2 };
```

Comment:
be careful here, because you are inserted a definition on the left of other passed definitions, for now it is well because we don't have this code in production, in the future it can't happen because is incompatible.

Reply:
ok